pallets/parachain-staking/src/lib.rs

```diff
@@ -2042,7 +2129,9 @@ pub mod pallet {
 			);
 
 			frame_system::Pallet::<T>::register_extra_weight_unchecked(
-				0, // TODO: T::WeightInfo::new_session(candidates_len_before as u32, removed as u32),
+				/* TODO: T::WeightInfo::new_session(candidates_len_before as u32,
+				 * removed as u32), */
+				T::DbWeight::get().reads(1),
```

Comment:
Not sure if this is sufficient.

Reply:
We only need to register the weight we produce inside this function. And that's only one DB read. So this is fine. 👍🏻